client/src/visualiser-src/avl-tree-visualiser/util/codeSnippets.ts

```diff
@@ -0,0 +1,84 @@
+export const insertCodeSnippet =
+`if (root == NULL) {
+  root = create_node(val);
+  return;
+}
+while (curr != NULL) {
+  if (val < curr->val) {
+    if (curr->left == NULL) {
+      curr->left = create_node(val);
+      return;
+    }
+    curr = curr->left;
+  } else {
+    if (curr->right == NULL) {
+      curr->right = create_node(val);
+      return;
+    }
+    curr = curr->right;
+  }
+}
+LHeight = height(root->left);
+RHeight = height(root->right);
+if (LHeight - RHeight > 1) {
+  if (val > root->left->val) {
+    root->left = rotate_left(root->left);
+  }
+  root = rotate_right(root);
+} else if (RHeight - LHeight > 1) {
+  if (val < root->right->val) {
+    root->right = rotate_right(root->right);
+  }
+  root = rotate_left(root);
+}
+return root;`;
+
+export const rotateLeftCodeSnippet =
```

Comment:
can remove the rotate left and right code snippets now since we don't have those as standalone operations for avl